TriggerRelease/index.ts

```diff
@@ -0,0 +1,93 @@
+import tl = require('vsts-task-lib/task');
+import url = require('url');
+
+import {EFClient} from "ef-client";
+
+let parseParameters = function(params: string) {
```

Comment:
How about moving this into the shared library?

Reply:
Done